### ISSUE TITLE
Steal how-can-i-use-xdebug-with-civicrm-unit-tests from SE

### ISSUE THREAD

**Issue by michaelmcandrew** [civicrm/civicrm-dev-docs#123]
See http://civicrm.stackexchange.com/questions/15896/how-can-i-use-xdebug-with-civicrm-unit-tests/15899

**Comment by totten** [comment]
Someone looking at this thread of out context might misunderstand the subject line. I wrote the original text in StackExchange. I often write docs for CiviCRM in different media as questions/topics come up (Mattermost, Stackexchange, Github, wiki, etc). As the original author, I wholeheartedly embrace migrating/consolidating these into `civicrm-dev-docs.git`. Michael and I are regular collaborators here in this documentation. When reading this, I understood the title to be a little cheeky -- and entirely consistent with our collaborative efforts to develop documentation. Anyone who says otherwise is concern-trolling.

**Comment by michaelmcandrew** [comment]
Indeed. Migrating content from CC licensed http://civicrm.stackexchange.com/ to https://docs.civicrm.org seems appriopriate and not controversial - was just trying to liven up the issue queue titles :)
